README.md

```diff
@@ -3,7 +3,7 @@ About pygithub
 
 Home: http://pygithub.github.io/PyGithub/v1/
 
-Package license: GNU Library or Lesser General Public License (LGPL)
+Package license: GPL-3.0
```

Comment:
So, this is incorrect as it is not under GPL-3. It is instead under LGPL-3. It seems this was incorrectly changed in PR ( https://github.com/conda-forge/pygithub-feedstock/pull/6 ). Hence why it appears in the re-rendering. Please see PR ( https://github.com/conda-forge/pygithub-feedstock/pull/10 ) for a correction.